docs/container-contract.md

```diff
@@ -4,32 +4,31 @@ linkTitle: "Container Contracts"
 weight: 8
 ---
 -->
-# Container Contract
+# Container Contracts
 
-Each container image used as a step in a [`Task`](tasks.md) must comply with a
-specific contract.
+Each container image that executes a `Step` in a [`Task`](tasks.md) must
+comply with the container contract described in this document.
 
-## Entrypoint
+When a `Task` instantiates and runs containers that execute the `Steps` in the `Task`,
+each container's `entrypoint` is overwritten with a custom binary that ensures the
+containers within the `Task's` Pod are executed in the order specified in the `Task`
+definition. Because of this, you must always specify a `command` value.
```

Comment:
It's not a `must`, but it's recommended.

Something that was not in this docs originally but that it's perhaps worth mentioning, is that when `script` mode is used, a command must not be specified, e.g.:

```
        - name: setup-comment
          image: python:3-alpine
          script: |
            #!/usr/bin/env python
            import json
            (...)
```

Docs about `script`: https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#running-scripts-within-steps

Reply:
Added this information to the doc.